src/OpenCensus.Exporter.ApplicationInsights/Implementation/TraceExporterHandler.cs

```diff
@@ -35,235 +36,238 @@ public TraceExporterHandler(TelemetryConfiguration telemetryConfiguration)
             this.telemetryClient = new TelemetryClient(telemetryConfiguration);
         }
 
-        public void Export(IEnumerable<ISpanData> spanDataList)
+        public async Task Export(IEnumerable<ISpanData> spanDataList)
         {
-            foreach (var span in spanDataList)
+            await Task.Run(async () =>
```

Comment:
can you please confirm this is the only change in this file. GitHub failed to compare nicely

Reply:
Correct, I have re-verified that this is the only change in this file.